x/chat/client/cli/query.go

```diff
@@ -24,8 +25,48 @@ func GetQueryCmd(queryRoute string) *cobra.Command {
 		RunE:                       client.ValidateCmd,
 	}
 
-	// this line is used by starport scaffolding # 1
+	cmd.AddCommand(
+		CmdDescribeChannel(),
+	)
 
-	return cmd 
+	return cmd
 }
 
+// CmdDescribeChannel returns the command to describe a channel
+func CmdDescribeChannel() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "describe-channel [channel-id]",
```

Comment:
"Describe" sounds like action with side-effects. Let's stick to CRUD-like verbs: show, create, update, etc.